api/kyverno/v2alpha1/policy_exception_types.go

```diff
@@ -71,6 +71,23 @@ type PolicyExceptionSpec struct {
 
 	// Exceptions is a list policy/rules to be excluded
 	Exceptions []Exception `json:"exceptions"`
+
+	// PodSecurity is used to check if resource applies to the exception
+	// Applicable only to validate.podSecurity.
+	PodSecurity PodSecuritySpec `json:"podSecurity,omitempty"`
```

Comment:
You can use [kyvernov1.PodSecurityStandard](https://github.com/kyverno/kyverno/blob/main/api/kyverno/v1/common_types.go#L420) instead defining a new one.

Reply:
Will do, Thanks.